nova/scheduler/filters/shard_filter.py

```diff
@@ -131,21 +131,6 @@ def host_passes(self, host_state, spec_obj):
                         'shard_prefix': self._SHARD_PREFIX})
             return False
 
-        # forbid changing the shard of an instance
```

Comment:
Do we now allow to change shard/vcenter of an instance during migration ?

Reply:
Yes, that is part of the change. I'll add it to the commit message